CMakeLists.txt

```diff
@@ -125,24 +125,11 @@ if(NOT DEFINED CMAKE_CXX_STANDARD)
   endif()
 endif()
 
-if(WITH_ABSEIL)
-  find_package(absl CONFIG REQUIRED)
-
-  set(CORE_RUNTIME_LIBS absl::bad_variant_access absl::any absl::base
```

Comment:
Where is the definition of `CORE_RUNTIME_LIBS` moved to?

Reply:
The variable is actually removed, and these libraries are now directly linked to `opentelemetry-api`

![test](https://user-images.githubusercontent.com/1196320/161816291-a5cc6105-ede2-4150-a4e0-c2997e6d7971.JPG)